portion/dict.py

```diff
@@ -88,43 +90,44 @@ def find(self, value):
         :param value: value to look for.
         :return: an Interval instance.
         """
-        return Interval(*(i for i, v in self._items if v == value))
+        return Interval(*(i for i, v in self._storage.items() if v == value))
 
     def items(self):
         """
-        Return a sorted list of (Interval, value) pairs.
+        Return a set-like object providing a view on contained items.
 
-        :return: a sorted list of 2-uples.
+        :return: a set-like object.
         """
-        return sorted(self._items, key=_sort)
+        return self._storage.items()
 
     def keys(self):
         """
-        Return the list of underlying Interval instances.
+        Return a set-like object providing a view on existing keys.
 
-        :return: a list of intervals.
+        :return: a set-like object.
         """
-        return [i for i, v in self.items()]
+        return self._storage.keys()
 
     def values(self):
         """
-        Return the list of values.
+        Return a set-like object providing a view on contained values.
```

Comment:
I think the naming “set-like” is miss leading; in particular `.values()` is very likely to “contain” duplicates.

Reply:
I took the docstring from built-in dict, but I agree that the "set" thing can be misleading. Any suggestion?

Reply:
What about simply
```suggestion
        Return a view object on the contained values (see https://docs.python.org/3/library/stdtypes.html#dict-views).
```
?